File: src/sdk-reference/js/6/search-result/snippets/search-result.js
try {
  for (let i = 0; i < 5; i++) {
    await kuzzle.document.create('nyc-open-data', 'yellow-taxi', '', {
      category: 'suv'
    });
  }
  for (let i = 5; i < 15; i++) {
    await kuzzle.document.create('nyc-open-data', 'yellow-taxi', '', {
      category: 'limousine'
    });
  }
  await kuzzle.index.refresh('nyc-open-data');

  const response = await kuzzle.document.search('nyc-open-data', 'yellow-taxi', {
    query: {
      match: {
        category: 'suv'
      }
    }
  }, {
    from: 1,
    size: 2
  });

  const nextResponse = await response.next();

  console.log(`Successfully retrieved ${nextResponse.fetched} documents`);
  console.log(nextResponse);
} catch (error) {
  console.error(error.message);
}

File: src/sdk-reference/csharp/1/search-result/next/snippets/search-result.cs
try {
  for (uint i = 0; i < 5; i++) {
    kuzzle.document.create("nyc-open-data", "yellow-taxi", "", @"{
      ""category"": ""suv""
    }");
  }
  for (uint i = 5; i < 15; i++) {
    kuzzle.document.create("nyc-open-data", "yellow-taxi", "", @"{
      ""category"": ""limousine""
    }");
  }
  kuzzle.index.refresh("nyc-open-data");

  query_options options = {};
  options.scroll = "1m";
  options.size = 2;

  SearchResult first_page = kuzzle.document.search(
    "nyc-open-data",
    "yellow-taxi",
    @"{
      ""query"": {
        ""match"": {
          ""category"": ""suv""
        }
      }
    }",
    options);

  SearchResult second_page = first_page.next();

  Console.WriteLine("Successfully retrieved " + second_page.fetched + " documents");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}
